### PR TITLE
fix(suite): #10877 timerange dropdown border

### LINKED ISSUE #10877: Empower/Range input has extra gray container + old style of inputs
image

### PR DESCRIPTION
**Pull request by martykan**
## Description

Fix borders and margin in Timerange component

## Related Issue

Resolve #10877

## Screenshots:

Screenshot 2024-02-14 at 16 28 51

Screenshot 2024-02-14 at 16 28 44

### CODE REVIEW

File: packages/components/src/components/Timerange/Timerange.tsx [hunk 1]
```diff
@@ -420,6 +420,7 @@ const StyledTimerange = styled.div`
     flex-direction: column;
     background: ${({ theme }) => theme.backgroundSurfaceElevation1};
     border-radius: ${borders.radii.sm};
+    margin: -${spacingsPx.sm};
```
Comment (jvaclavik) [inline packages/components/src/components/Timerange/Timerange.tsx:423]: Can we do it without a negative margin? 👀

**Comment by jvaclavik** [comment]
Btw I spot this issue. Can you also see it?

Screenshot 2024-02-19 at 10 16 46


And what about rounded corners for inputs?

Screenshot 2024-02-19 at 10 32 41

I would use probably `borders.radii.xs`

**Comment by martykan** [comment]
The strange borders are gone as per the screenshots above. I agree rounded corners would be nicer

File: packages/components/src/components/Timerange/Timerange.tsx (hunk 1 above)
Comment (martykan) [reply packages/components/src/components/Timerange/Timerange.tsx:423]: Yes, well this was the quickest way 😃 
The space comes from padding in the Menu component, which is used in Dropdown, so we would need to add props there to allow overriding it.

Comment (jvaclavik) [reply packages/components/src/components/Timerange/Timerange.tsx:423]: I see, then let's skip keep negative margin

**Comment by jvaclavik** [comment]
> The strange borders are gone as per the screenshots above. I agree rounded corners would be nicer

Could you please add these rounded corners?

**Comment by martykan** [comment]
Updated

Screenshot 2024-02-19 at 11 43 23

**Review by jvaclavik — APPROVED** [review]

**Pull request merged**